### PR TITLE
Fixed docs...

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

File: dev/podman-kind.md [hunk 1]
```diff
@@ -14,10 +14,15 @@ brew install podman
...
 Once done, init and start the machine:
 ```
-podman machine init
+podman machine init --volume $HOME:$HOME
 podman machine start
 ```
 
+## Install system helper service
+```
+sudo /usr/local/Cellar/podman/4.0.3/bin/podman-mac-helper install
```
Comment (kuujo) [inline dev/podman-kind.md:23]: We should note that this is only a one-time thing

Comment (kuujo) [reply dev/podman-kind.md:23]: Also, the user needs to run the other `export` command to set `$DOCKER_HOST` correct?

Comment (tomikazi) [reply dev/podman-kind.md:23]: The command is an alternative to DOCKER_HOST export.

Comment (kuujo) [reply dev/podman-kind.md:25]: Ahh I didn't see that... sounds good

**Review by kuujo — APPROVED** [review]

**Review by kuujo — APPROVED** [review]

**Pull request merged**